assignment/api/src/resources/assignment.properties

```diff
@@ -1179,8 +1179,8 @@ email.reminder.andhours=and {0} hours.
 email.reminder.hours={0} hours.
 email.reminder.days={0} days
 
-defaultGrade.notGrade=a Grade
-defaultGrade.notSubmit=Not Submit
+defaultGrade.notGrade=A Grade
+defaultGrade.notSubmit=A Submission
```

Comment:
I'm not sure about this new translation... but I'm not an English speaker.

Reply:
This is a good English translation.